### ISSUE TITLE
v-slot' directive doesn't support any modifier.

### ISSUE THREAD

**Issue by Rony20191** [vuejs/eslint-plugin-vue#1272]
**Tell us about your environment**
* **ESLint version:** ^6.7.2
* **eslint-plugin-vue version:** ^6.2.2
* **Node version:**  v12.16.0

**Please show your full configuration:**

```json

```

**What did you do?**	

```vue

<template>
  <v-card>
    <v-card-text>
      <v-data-table
        :search="table.search"
        :headers="table.headers"
        :items="table.dados"
        :items-per-page="5"
      >
        <template v-slot:top>
          <v-row dense align="center" justify="center">
            <span id="titulo" style="text-aling:center;">{{title}}</span>
          </v-row>

          <v-row dense>
            <v-col cols="12" md="4" sm="12">
              <v-text-field class="ml-2" v-model="table.search" append-icon="mdi-magnify"></v-text-field>
            </v-col>
          </v-row>
          <v-row dense>
            <v-spacer />

            <v-btn class="mr-4" @click="$refs.dialogTelefone.show(table.dados)" color="primary">
              <v-icon left>mdi-plus</v-icon>E-Mail
            </v-btn>
          </v-row>
        </template>

        <template v-slot:item.principal="{item}">
          <v-icon
            :color="item.principal?'success':'error'"
          >{{item.principal?'mdi-check':'mdi-close'}}</v-icon>
        </template>

        <template v-slot:item.actions="{item}">
          <v-btn @click="alterar(item)" icon>
            <v-icon>mdi-pencil</v-icon>
          </v-btn>
          <v-btn @click="remove(item)" icon>
            <v-icon>mdi-delete</v-icon>
          </v-btn>
        </template>
      </v-data-table>
    </v-card-text>
    <DialogEndereco ref="dialogEndereco" @OnSaved="adicionar($event)" />
  </v-card>
</template>


```

**What did you expect to happen?**

One week it worked perfectly, now it is notifying this error.

**What actually happened?**

[vue/valid-v-slot]
An element cannot have multiple '<template>' elements which are distributed to the same slot.

[vue/valid-v-slot]
'v-slot' directive doesn't support any modifier.

**Comment by ota-meshi** [comment]
Hi. It is a duplicate of #1165. Please track #1165.

**Comment by FloEdelmann** [comment]
If this is a problem that appears in the latest version of eslint-plugin-vue, please open a new issue following the template, rather than commenting on an issue that was closed 2.5 years ago.
